subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -1327,14 +1327,25 @@ static inline u32_t isr_rx_scan(u8_t devmatch_ok, u8_t devmatch_id,
 		       access_addr[0], &conn->access_addr[0], 4);
 		memcpy(&pdu_adv_tx->payload.connect_ind.lldata.crc_init[0],
 		       &conn->crc_init[0], 3);
-		pdu_adv_tx->payload.connect_ind.lldata. win_size = 1;
+		pdu_adv_tx->payload.connect_ind.lldata.win_size = 1;
 
 		conn_interval_us =
 			(u32_t)_radio.scanner.conn_interval * 1250;
 
 		conn_offset_us = radio_tmr_end_get() + 502 + 1250;
-		conn_offset_us -= radio_tx_chain_delay_get(0, 0);
-		conn_offset_us -= radio_tx_ready_delay_get(0, 0);
+
+		/* Compensation for negative remainder values,
+		 * refer to radio_tmr_start in hal/radio.c
```

Comment:
The ticker is designed to, on its own, average the intervals between expiry to meet the BT Spec. definitions of event intervals. This means, using a low power clock of 32KHz, its a jitter of +/- (30.517 / 2) us. The radio_tmr_* interface then uses the instantaneous jitter value to provide microsecond resolution (fine expiry). 

I could have a new interface, radio_tmr_start_adjust(u32_t *ticks_in_out, u32_t *remainder_in_out). ?